Code/GraphMol/RascalMCES/RascalOptions.h

```diff
@@ -58,6 +58,12 @@ struct RDKIT_RASCALMCES_EXPORT RascalOptions {
   bool ignoreAtomAromaticity = true; /* If true, atoms are matched just on
                                         atomic number; if false, will treat
                                         aromatic and aliphatic as different. */
+  int minCliqueSize = -1; /* Normally, the minimum clique size is specified
```

Comment:
If the condition for this to be specified is that `minCliqueSize > 0`, then can't this be an unsigned int with a default value of 0?